tests/testsuite/check_cfg.rs

```diff
@@ -0,0 +1,463 @@
+use cargo_test_support::{basic_manifest, is_nightly, project};
```

Comment:
Can you add an inner doc comment that documents what this module is for?

Reply:
Done.